csrankings-h.csv

```diff
@@ -416,6 +417,7 @@ Hasan M. Jamil,University of Idaho,http://www.uidaho.edu/engr/departments/cs/our
 Hasan Metin Aktulga,Michigan State University,http://www.cse.msu.edu/~hma,NOSCHOLARPAGE
 Hasan Sözer,Özyeğin University,https://faculty.ozyegin.edu.tr/hsozer,L3FuUX0AAAAJ
 Hasan Ural,University of Ottawa,http://www.site.uottawa.ca/~ural,NOSCHOLARPAGE
+Hasnain Heickal,University of Dhaka,https://hheickal.github.io,oQc7yEAAAAAJ

```

Comment:
Faculty with a <75% appointment are not eligible for inclusion (except for sabbaticals).